selfdrive/common/statlog.cc

```diff
@@ -17,6 +17,8 @@ class StatlogState : public LogState {
 static StatlogState s = {};
 
 static void log(const char* metric_type, const char* metric, const char* fmt, ...) {
+  if (!s.initialized) s.initialize();
```

Comment:
You want to take the lock like in `cloudlog_common`, this can cause race conditions if multiple threads call `log` at the same time.